CHANGELOG.md

```diff
@@ -7,13 +7,18 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+- Override the process environment variables with the remote equivalents by setting `MIRRORD_OVERRIDE_ENV`, like `MIRRORD_OVERRIDE_ENV="DB=fakedb;CONFIG=fake.config"`;
```

Comment:
We're assuming they'd want to override all for 80% of the cases, but maybe let the same variable accept different values for all, whitelist, blacklist?
So e.g.:
MIRRORD_OVERRIDE_ENV: true
or
MIRRORD_OVERRIDE_ENV: PATH, HOME
or
MIRRORD_OVERRIDE_ENV: !(PATH, HOME)

Reply:
It now loads all env vars, and the user specifies a list to filter out (reverse of what it was doing).

Reply:
I don't think we have a reason to believe an "exclude list" use case would be more common than an "include list" use case, I think we might as well support both.
btw I support using the same env var for all of them (since you can't have override_env = false && filter = [x, y, z]).

Reply:
I think we support both now, and I've added a panic if exclude/include share values.